.travis.yml

```diff
@@ -1,9 +1,11 @@
 sudo: false
 language: scala
+dist: trusty
```

Comment:
Due to: 
https://travis-ci.community/t/expected-feature-release-number-in-range-of-9-to-12-but-got-8-installing-oraclejdk8/1345